frontend/views/pages/GroupSettings.vue

```diff
@@ -146,8 +159,17 @@ export default ({
         contractID: this.$store.state.currentGroupId,
         key: 'groupPicture'
       }
+    },
+    isGroupAdmin () {
+      // TODO: group admin doesn't mean group creator
+      // https://github.com/okTurtles/group-income/issues/202
+      return this.groupSettings.groupCreator === this.ourUsername
```

Comment:
For now please comment this out and `return false` always.

The reason is we need to also check whether the Chelonia server admin has allowed us to create public chatrooms.

Reply:
```
isGroupAdmin () {
  // TODO: https://github.com/okTurtles/group-income/issues/202
  return false
}
```
Is this fine?